### PR TITLE
feat(blog): add blog and API packages

### PR DESCRIPTION
**Pull request by sauldeleon**
- Add next js blog package
- Add API package
- Updated dependencies across the whole monorepo
- renamed packages. Now the monorepo is under `@yisuses` scope and each package under `@whe` scope

### CODE REVIEW

**Comment by paulaguijarro** [comment]
sad-gopher

**Review by paulaguijarro — APPROVED** [review]

File: packages/common/jest.config.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import packageJson from './package.json'
-const packageName = packageJson.name.split('@yisuses/').pop()
+const packageName = packageJson.name.split('@whe/').pop()
```
Comment (bgonp) [inline packages/common/jest.config.ts:2]: Take a look to comment on packages/admin/jest.config.ts file

File: tsconfig.build.json [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
+    "noFallthroughCasesInSwitch": true,
+    "outDir": "./dist",
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5"
   },
   "exclude": ["node_modules", "dist"]
 }
```
Comment (bgonp) [inline tsconfig.build.json:21]: God bless alphabetical order

**Review by bgonp — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -23,3 +25,6 @@ dist
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# vercel
+.vercel
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply .gitignore:30]: this is included by default by the `create next app` script. I do not know if it is neccesary, but I can remove it for sure. Thanks!

File: packages/api/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import express, { Application, Request, Response } from 'express'
+
+const app: Application = express()
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/api/src/index.ts:3]: Done. Thanks!

File: packages/api/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "esModuleInterop": true,
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "outDir": "build",
+    "sourceMap": true,
+    "target": "es6"
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/api/tsconfig.json:10]: I have inherited the base configuration. I have removed the duplicated properties. I think its better that way

[the remark this answers is not in the document]
Comment (bgonp) [reply packages/api/tsconfig.json:10]: ahhhhh ok ok

File: packages/api/package.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  "main": "src/index.ts",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "tsc": "tsc",
+    "build": "tsc",
+    "start": "node --loader ts-node/esm src/index.ts",
+    "start:prod": "node build/index.js",
+    "dev": "nodemon --watch src -e ts --exec yarn run start",
+    "clean": "rimraf dist/*",
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/api/package.json:14]: Done and working like a charm. Thanks!

File: .gitignore (hunk 3 above)
[the remark this answers is not in the document]
Comment (sauldeleon) [reply .gitignore]: Done!

File: packages/api/tsconfig.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/api/tsconfig.json]: Changes applied

File: packages/admin/jest.config.ts [hunk 7]
```diff
@@ -3,7 +3,7 @@ import { pathsToModuleNameMapper } from 'ts-jest/utils'
 import { compilerOptions } from './tsconfig.paths.json'
 
 import packageJson from './package.json'
-const packageName = packageJson.name.split('@yisuses/').pop()
+const packageName = packageJson.name.split('@whe/').pop()
```
[the remark this answers is not in the document]
Comment (sauldeleon) [reply packages/admin/jest.config.ts]: Done!

File: packages/common/jest.config.ts (hunk 1 above)
Comment (sauldeleon) [reply packages/common/jest.config.ts]: Done!

**Review by bgonp — APPROVED** [review]
Everytime you force a commit, God kills a kitty

**Pull request merged**